src/oas.md

```diff
@@ -44,6 +44,27 @@ Each template expression in the path MUST correspond to a path parameter that is
 
 The value for these path parameters MUST NOT contain any unescaped "generic syntax" characters described by [RFC3986](https://tools.ietf.org/html/rfc3986#section-3): forward slashes (`/`), question marks (`?`), or hashes (`#`).
 
+The path templating is defined by the following [ABNF](https://tools.ietf.org/html/rfc5234) syntax
+
+```abnf
+path-template                  = "/" *( path-segment "/" ) [ path-segment ]
+path-segment                   = 1*( path-literal / template-expression )
+path-literal                   = 1*pchar
+template-expression            = "{" template-expression-param-name "}"
+template-expression-param-name = 1*( %x00-79 / %x7C / %x7E-10FFFF ) ; every UTF8 character except { and }
+
+pchar               = unreserved / pct-encoded / sub-delims / ":" / "@"
+unreserved          = ALPHA / DIGIT / "-" / "." / "_" / "~"
+pct-encoded         = "%" HEXDIG HEXDIG
+sub-delims          = "!" / "$" / "&" / "'" / "(" / ")"
+                    / "*" / "+" / "," / ";" / "="
+ALPHA               = %x41-5A / %x61-7A   ; A-Z / a-z
```

Comment:
Nitpick (ALPHA, DIGIT, HEXDIG) are part of RFC 5234 (ABNF) syntax itself and are usually dropped from final ABNFs. Do we want to align with `runtime expression` ABNF and not explicitly define those?

Reply:
Thanks for pointing this out! Removed since 3986 doesn't redefine them either.